scripts.py

```diff
@@ -691,3 +695,43 @@ def do_run(self, _db=None, cmd_args=None, output=sys.stdout, emailer_class=Email
         # Since the emailer didn't raise an exception we can assume we sent
         # the email successfully.
         _db.commit()
+
+
+class AlembicMigrateDatabase(Script):
+    @classmethod
+    def arg_parser(cls):
+        parser = argparse.ArgumentParser(
+            prog="Alembic Database Migration",
+            description="By default, running this script without any arguments "
+            "will create the tables and stamp as head if the database is new. Otherwise "
+            "it will perform an alembic 'upgrade head' command.",
+        )
+        parser.add_argument(
+            "-d",
+            "--downgrade",
+            help="Downgrade to a specific version.",
+            required=False,
+            default=None,
+        )
+        parser.add_argument(
+            "-u",
+            "--upgrade",
+            help="Upgrade to a specific version.",
+            required=False,
+            default="head",
+        )
+        return parser
+
+    def do_run(self, cmd_args=None):
+        args = self.parse_command_line(cmd_args=cmd_args)
+        config = AlembicConfig("alembic.ini")
+
+        try:
```

Comment:
Maybe this targeted migration logic can move to `db_migration.migrate` since it's a lot of similar code between the two methods.